db/schema.rb

```diff
@@ -10,4 +24,12 @@
     t.datetime "updated_at",             null: false
   end
 
+  create_table "skills", force: :cascade do |t|
```

Comment:
Be careful about adding in files that don't directly relate to the PR into the PR code. Based on the changes I saw, this shouldn't be in here.